python/test/unit/mesh/test_mesh_partitioners.py

```diff
@@ -20,13 +20,37 @@
 assert (tempdir)
 
 
-@pytest.mark.parametrize("partitioner", [partition_cells_graph])
+partitioners = [dolfinx.cpp.graph.partitioner()]
```

Comment:
Should be empty list? Existence of default partitioner covered in next try/except statement.